dcos/dcos-ui
# DCOS-46247 - COPS-4205: No tasks show for a service

- update @dcos/mesos-client to 0.2.6
- update @dcos/http-service to 1.1.0

- Closes DCOS-46247

## Testing

- `npm i`
- See [COPS-4205](https://jira.mesosphere.com/browse/COPS-4205)
- make sure that /mesos/api/v1 endpoints have timestamp part `_ts=<number>`
